lambdas/cfn-set-bucket-tags/README.md

```diff
@@ -0,0 +1,89 @@
+# cfn-set-bucket-tags
+
+The `SetBucketTags` sets the OwnerEmail of a bucket
+
+Inventory of source code and supporting files:
+
+- set_bucket_tags - Code for the application's Lambda function.
+- events - Invocation events that you can use to invoke the function.
+- tests - Unit tests for the application code.
+- template.yaml - A template that defines the application's AWS resources.
+
+## Use in a Cloudformation Template
+Insert in your Cloudformation template a block similar to this as a sibling
+of the S3 bucket (called in this example `S3Bucket`) :
+```yaml
+  'Fn::Transform':
+    Name: "SetBucketTags"
+    Parameters:
+      BucketName: !Ref S3Bucket
+```
+
+## Deploy the sample application
+
+The Serverless Application Model Command Line Interface (SAM CLI) is an
+extension of the AWS CLI that adds functionality for building and testing Lambda
+applications. It uses Docker to run your functions in an Amazon Linux
+environment that matches Lambda. It can also emulate your application's build
+environment and API.
+
+## Use the SAM CLI to build and test locally
+
+Build your application with the `sam build --use-container` command.
+
+```bash
+cfn-set-bucket-tags$ sam build --use-container
```

Comment:
is `cfn-set-bucket-tags$` part of the command to build?

Reply:
No, this is the prompt, and it's in the README you get from running `sam init`.